### PR TITLE
staff_in_lab_count

### PR DESCRIPTION
**Pull request by tahabi**
with reference to https://github.com/ocf/labstats/pull/2

### CODE REVIEW

File: ocflib/lab/stats.py [hunk 1]
```diff
@@ -66,6 +66,13 @@ def staff_in_lab():
         return [Session.from_row(r) for r in c]
 
 
+def staff_in_lab_count():
+    """Return the count of unique staff in the lab"""
+    # thank SO: http://stackoverflow.com/questions/10024646/how-to-get-unique-list-using-a-key-word-python
+    seen = set()
+    return len([seen.add(staff.user) or staff for staff in staff_in_lab() if staff.user not in seen])
```
Comment (chriskuehl) [inline ocflib/lab/stats.py]: this is insane, please don't commit this

can't you just write `return len({session.user for session in staff_in_lab()})`?

Comment (tahabi) [inline ocflib/lab/stats.py]: lol that is a much better solution

File: ocflib/lab/stats.py [hunk 2]
```diff
@@ -66,6 +66,12 @@ def staff_in_lab():
         return [Session.from_row(r) for r in c]
 
 
+def staff_in_lab_count():
+    """Return the count of unique staff in the lab"""
+    # thank based ckuehl https://github.com/ocf/ocflib/pull/47#discussion_r97914296
```
Comment (kpengboy) [inline ocflib/lab/stats.py]: I would say the comment would be superfluous in the long run...

Comment (tahabi) [inline ocflib/lab/stats.py]: one must atone for one's sins

**Pull request merged**